testing/unit_tests/test_transforms.py

```diff
@@ -238,6 +238,20 @@ def test_NormalizeInstance(im_seg):
     assert abs(do_tensor.mean() - 0.0) <= 1e-2
     assert abs(do_tensor.std() - 1.0) <= 1e-2
 
+    # Transform on Numpy - Uniform sample
+    im = np.ones(im[0].shape)
+    do_im, _ = transform(im.copy(), metadata_in)
+    # Check mean-substraction
+    assert abs(np.mean(do_im) - 0.0) <= 1e-2
```

Comment:
I don't think I understand this, why should this condition `np.mean(do_im) - 0.0` be checked when the input is all ones? I think I'm missing the point of the test here

Reply:
The new tests are there to cover the new "if sample uniform: do mean-subtraction" part of NormalizeInstance, whereas the old tests cover the "else: normalize sample" part.
So here, we checked if the "if" part worked properly i.e. after the transform (mean-substraction), the "ones" should all be "zeros".
Let me know if I interpreted your question correctly.

Reply:
But, say, `im` is a 3x3 matrix of ones, then `np.mean(im)` is 1.0 right? Then the assert statement will never be true? Does it make sense?

Reply:
Ah wait, it is checking the mean of `do_im` and not `im`, got it! Sorry for the confusion!

Reply:
Ok! thanks @naga-karthik for the review :)